pack_savegame.py

```
@@ -8,30 +8,87 @@
 payload = f.readlines()
 f.close()
 
+# Read base64 encoded font
+import os
```

Ruff (0.14.3)

Redefinition of unused `os` from line 4

Remove definition: `os`

(F811)